google/cloud/bigtable/examples/bigtable_instance_admin_snippets.cc

```diff
@@ -272,17 +284,18 @@ void CreateCluster(google::cloud::bigtable::InstanceAdmin instance_admin,
 
   //! [create cluster] [START bigtable_create_cluster]
   namespace cbt = google::cloud::bigtable;
+  using google::cloud::future;
   using google::cloud::StatusOr;
   [](cbt::InstanceAdmin instance_admin, std::string instance_id,
      std::string cluster_id, std::string zone) {
     auto cluster_config = cbt::ClusterConfig(zone, 3, cbt::ClusterConfig::HDD);
-    auto future_cluster = instance_admin.CreateCluster(
-        cluster_config, cbt::InstanceId(instance_id),
-        cbt::ClusterId(cluster_id));
+    future<StatusOr<google::bigtable::admin::v2::Cluster>> future_cluster =
```

Comment:
Nit: Other examples above were like `f_cluster`.

Reply:
Good point. I settled for `foo_future` everywhere, more explicit is better than less explicit for these examples.